orchagent/crmorch.cpp

```diff
@@ -928,6 +939,16 @@ void CrmOrch::getResAvailableCounters()
                 for (auto &cnt : res.second.countersMap)
                 {
                     sai_status_t status = sai_acl_api->get_acl_table_attribute(cnt.second.id, 1, &attr);
+                    if ((status == SAI_STATUS_NOT_SUPPORTED) ||
+                        (status == SAI_STATUS_NOT_IMPLEMENTED) ||
+                        SAI_STATUS_IS_ATTR_NOT_SUPPORTED(status) ||
+                        SAI_STATUS_IS_ATTR_NOT_IMPLEMENTED(status))
+                    {
+                        // mark unsupported resources
+                        res.second.resStatus = CrmResourceStatus::CRM_RES_NOT_SUPPORTED;
+                        SWSS_LOG_NOTICE("CRM resource %s not supported", crmResTypeNameMap.at(res.first).c_str());
```

Comment:
Please confirm this log won't be spewed periodically. If so, please reduce to INFO level.

Reply:
It will be printed only once since if the resource is marked as not supported, it will be skipped in further checks:
https://github.com/sonic-net/sonic-swss/blob/master/orchagent/crmorch.cpp#L845